registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -1,598 +1,681 @@
+import io.mosip.registration.processor.core.packet.dto.abis.AbisIdentifyRequestDto;
+import io.mosip.registration.processor.core.packet.dto.abis.AbisIdentifyRequestGalleryDto;
+import io.mosip.registration.processor.core.packet.dto.abis.AbisInsertRequestDto;
+import io.mosip.registration.processor.core.packet.dto.abis.AbisRequestDto;
+import io.mosip.registration.processor.core.packet.dto.abis.Flag;
+import io.mosip.registration.processor.core.packet.dto.abis.ReferenceIdDto;
+import io.mosip.registration.processor.core.packet.dto.abis.RegBioRefDto;
+import io.mosip.registration.processor.core.packet.dto.abis.RegDemoDedupeListDto;
+import io.mosip.registration.processor.core.spi.packetmanager.PacketInfoManager;
+import io.mosip.registration.processor.core.spi.restclient.RegistrationProcessorRestClientService;
+import io.mosip.registration.processor.core.status.util.StatusUtil;
+import io.mosip.registration.processor.core.status.util.TrimExceptionMessage;
+import io.mosip.registration.processor.core.util.JsonUtil;
+import io.mosip.registration.processor.packet.storage.dto.ApplicantInfoDto;
+import io.mosip.registration.processor.packet.storage.utils.BIRConverter;
+import io.mosip.registration.processor.packet.storage.utils.PriorityBasedPacketManagerService;
+import io.mosip.registration.processor.packet.storage.utils.Utilities;
+import io.mosip.registration.processor.rest.client.audit.builder.AuditLogRequestBuilder;
+import io.mosip.registration.processor.status.dto.InternalRegistrationStatusDto;
+import io.mosip.registration.processor.status.dto.RegistrationStatusDto;
+import io.mosip.registration.processor.status.service.RegistrationStatusService;
+
+/**
+ * The Class AbisHandlerStage.
+ * 
+ * @author M1048358 Alok
+ */
+@Service
+public class AbisHandlerStage extends MosipVerticleAPIManager {
+
+	/** The cluster manager url. */
+	@Value("${vertx.cluster.configuration}")
+	private String clusterManagerUrl;
+
+	/** The max results. */
+	@Value("${registration.processor.abis.maxResults}")
+	private String maxResults;
+
+	/** The target FPIR. */
+	@Value("${registration.processor.abis.targetFPIR}")
+	private String targetFPIR;
+
+	/** server port number. */
+	@Value("${server.port}")
+	private String port;
+
+	/** worker pool size. */
+	@Value("${worker.pool.size}")
+	private Integer workerPoolSize;
+
+	/** After this time intervel, message should be considered as expired (In seconds). */
+	@Value("${mosip.regproc.abis.handler.message.expiry-time-limit}")
+	private Long messageExpiryTimeLimit;
+
+	@Value("${registration.processor.policy.id}")
+	private String policyId;
+
+	@Value("${registration.processor.subscriber.id}")
+	private String subscriberId;
+
+	@Value("#{${mosip.regproc.abis.handler.biometric-modalities-segments-mapping}}")
+	private Map<String, List<String>> biometricModalitySegmentsMap;
+	
+	@Value("${mosip.regproc.abis.handler.status.code:REJECTED}")
+	private String statusCode;

```

Comment:
both config and variable name lets change to covey what and when this variable will be used. like insufficentBiometricsStatusCode

Reply:
fixed